app/pages/project/instances/instance/tabs/MetricsTab.tsx

```diff
@@ -54,27 +64,77 @@ function DiskMetric({
     { placeholderData: (x) => x }
   )
 
-  const data = (metrics?.items || []).map(({ datum, timestamp }) => ({
-    timestamp: timestamp.getTime(),
-    // all of these metrics are cumulative ints
-    value: (datum.datum as Cumulativeint64).value,
-  }))
+  const isBytesChart = unit === 'Bytes'
+
+  const largestValue = useMemo(() => {
+    if (!metrics || metrics.items.length === 0) return 0
+    return Math.max(...metrics.items.map((m) => (m.datum.datum as Cumulativeint64).value))
+  }, [metrics])
```

Comment:
I fixed the last value not necessarily being the largest by looking at all values and finding the max. I always feel weird about spreading 3000 args into `Math.max` but I tested it a while ago (we do this elsewhere) and it is apparently fine.